pwa/src/templates/mappingDetailTemplate/MappingDetailsTemplate.tsx

```diff
@@ -50,7 +91,56 @@ export const MappingDetailTemplate: React.FC<MappingDetailsTemplateProps> = ({ m
             handleToggleDashboard={{ handleToggle: toggleFromDashboard, isActive: !!dashboardCard }}
           />
 
-          <MappingFormTemplate mapping={getMapping.data} />
+          <div className={styles.tabContainer}>
+            <TabContext value={currentTabs.mappingDetailTabs.toString()}>
+              <Tabs
+                value={currentTabs.mappingDetailTabs}
+                onChange={(_, newValue: number) => {
+                  setCurrentTabs({ ...currentTabs, mappingDetailTabs: newValue });
+                }}
+                variant="scrollable"
+              >
+                <Tab className={styles.tab} label={t("General")} value={0} />
+                <Tab className={styles.tab} label={t("Test Mappings")} value={1} />
+              </Tabs>
+
+              <TabPanel className={styles.tabPanel} value="0">
+                <MappingFormTemplate mapping={getMapping.data} />
+              </TabPanel>
+
+              <TabPanel className={styles.tabPanel} value="1">
+                <form onSubmit={handleSubmit(onSubmitTest)}>
+                  <Button
+                    className={clsx(styles.buttonIcon, styles.testConnectionButton)}
+                    disabled={isLoading.mappingForm}
+                    type="submit"
+                  >
+                    <FontAwesomeIcon icon={faArrowsRotate} />
+                    {t("Test simple mapping")}
+                  </Button>
+
+                  <div className={styles.content}>
+                    <FormField>
+                      <FormFieldInput>
+                        <FormFieldLabel>{t("Input")}</FormFieldLabel>
+                        <Textarea
+                          {...{ register, errors }}
+                          name="input"
+                          validation={{ validate: validateStringAsJSON }}
+                          disabled={isLoading.mappingForm}
+                        />
+                        {errors["input"] && <ErrorMessage message={errors["input"].message} />}
+                      </FormFieldInput>
+                    </FormField>
+                    <FormField>
+                      <FormFieldLabel>{t("Output")}</FormFieldLabel>
+                      <pre>{JSON.stringify(testSimpleMapping.data, null, 2)}</pre>
+                    </FormField>
```

Comment:
Right now, when nothing has been sent yet, we already see the string "Output", without anything underneath. I think we should do something about that, possible solutions:

- Hide the "Output" part, until there is output to be shown;
- Grey it out;
- Put another message underneath it, e.g. "Your mapping test output will be shown here."